doc/ref/xtndxmpl.xml

```diff
@@ -1329,7 +1329,7 @@ gap> InstallMethod( Random,
 >    "for full ring Z/nZ",
 >    [ CategoryCollections( IsMyZmodnZObjNonprime ) and IsWholeFamily ],
 >    R -> MyZmodnZObj( ElementsFamily( FamilyObj( R ) ),
->                    Random( [ 0 .. Size( R ) - 1 ] ) ) );
+>                    Random( 0 .. Size( R ) - 1 ) ) );
```

Comment:
I guess this is a typo? (The .. ).

Reply:
Indeed, ouch :-(. I'll fix it, thank you very much for careful checking.

Reply:
Fixed